reciprocalspaceship/dataset.py

```diff
@@ -131,6 +149,38 @@ def merged(self):
     def merged(self, val):
         self._merged = val
 
+    @property
+    @range_indexed
+    def hkls(self):
+        """Miller indices"""
+        hkl = self[["H", "K", "L"]].to_numpy(dtype=np.int32)
+        return hkl
+
+    def get_hkls(self):
+        """For backwards compatibility retain the get_hkls method in addition to the dataset.hkls attribute"""
```

Comment:
I think this docstring should be more of a method description, rather than a rationale for keeping the method

Reply:
Okay, I changed it. Let me know if you are okay with the update.